xinference/model/llm/utils.py

```diff
@@ -333,8 +334,9 @@ def _handle_qwen_tool_result(cls, text: str) -> List[Tuple]:
         for content in contents:
             content = content.strip()
             if content:
-                if content.startswith(QWEN_TOOL_CALL_SYMBOLS[0]):
-                    content = content[len(QWEN_TOOL_CALL_SYMBOLS[0]) :]
+                pos = content.find(QWEN_TOOL_CALL_SYMBOLS[0])
+                if pos != -1:
+                    content = content[pos + len(QWEN_TOOL_CALL_SYMBOLS[0]) :]
```

Comment:
Will this be compatible with qwen 1.5 and 2 tool calls?
What's the qwen2.5 tool call results look like?